e2e_testing/xfail_sets.py

```diff
@@ -800,6 +800,7 @@
     "ConvolutionModule2DTransposeStrided_basic",
     "ConvolutionModule2DTranspose_basic",
     "ElementwiseClampModule_basic",
+    "ElementwisePreluModule",
```

Comment:
This should be `"ElementwisePreluModule_basic"`

Reply:
Thanks, missed that.